engine/schema/src/main/java/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -364,6 +368,29 @@ protected void upgrade(CloudStackVersion dbVersion, CloudStackVersion currentVer
             }
         }
         updateSystemVmTemplates(upgrades);
+        executeViewScripts();
```

Comment:
Can you please switch line 370 and 371 so that the view upgrade happens before updating the system VM template?